developmentseed/pearl-frontend
# Disable AOI fill opacity

Fix #53. How to test:

- Run the app locally
- Open an existing project on local and staging website, colors on local should be more vivid as there is no overlay in the AOI
- Create a new AOI, fill should be present when drawing the AOI, but not after it is finished drawn
- Upload new AOI, fill should not be present after import

@geohacker ready for review.
